2023-11-26 09:19:13.344 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: Request timed out: HTTPSConnectionPool(host='api.openai.com', port=443): Max retries exceeded with url: /v1/completions (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x000001F0DB9BB390>, 'Connection to api.openai.com timed out. (connect timeout=600)'))
2023-11-26 10:58:03.305 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: Rate limit reached for text-davinci-003 in organization org-YfHz35c0f0VPg1kimN26kfBl on requests per day (RPD): Limit 200, Used 200, Requested 1. Please try again in 7m12s. Visit https://platform.openai.com/account/rate-limits to learn more. You can increase your rate limit by adding a payment method to your account at https://platform.openai.com/account/billing.
2023-11-26 11:06:48.113 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: Rate limit reached for text-davinci-003 in organization org-YfHz35c0f0VPg1kimN26kfBl on requests per day (RPD): Limit 200, Used 200, Requested 1. Please try again in 7m12s. Visit https://platform.openai.com/account/rate-limits to learn more. You can increase your rate limit by adding a payment method to your account at https://platform.openai.com/account/billing.
2023-11-26 13:51:32.947 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: No such organization: org-cODSJjftWgVspplR3MwUi2HN.
2023-11-26 13:55:54.554 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: No such organization: org-cODSJjftWgVspplR3MwUi2HN.
2023-11-26 14:27:01.469 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: This model's maximum context length is 4097 tokens, however you requested 4737 tokens (3237 in your prompt; 1500 for the completion). Please reduce your prompt; or completion length.
2023-11-26 14:28:42.490 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: This model's maximum context length is 4097 tokens, however you requested 4160 tokens (2660 in your prompt; 1500 for the completion). Please reduce your prompt; or completion length.
2023-11-26 18:55:58.562 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: No such organization: org-YfHz35c0f0VPg1kimN26kfBl.
2023-11-26 18:57:43.422 | ERROR    | doc_analyze.doc_analyze:run:16 - An error occurred during translation: No such organization: org-YfHz35c0f0VPg1kimN26kfBl.
